minio/api.py

```diff
@@ -34,6 +34,7 @@
 import certifi
 import urllib3
 from urllib3._collections import HTTPHeaderDict
+from tqdm import tqdm
```

Comment:
I see, I try to make silmilar with `put_object()`

Reply:
![스크린샷 2022-09-09 오후 11 18 05](https://user-images.githubusercontent.com/52662993/189371389-c673e177-2523-456c-8cd5-56e11d066472.png)
add put_object like progress

I think if this progress option is used many times in api.py

In my opinion, it would be better to seperate progress as function to some where, like helpers.py or something.

please give som feed back